package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.4.0",
```

Comment:
using https://github.com/gsoft-inc/craco that was made to specifically override config for create react app